File: C/StringLength.c
//Program to find the length of a string without using library function

#include<stdio.h>

int main()
{
	char name[10];
	int i=0;
	
	printf("Please enter your first name: ");
	scanf("%s", name);
	
	while(name[i]!='\0')
	{
		i++;
	}
	
	printf("Your first name has %d characters.\n",i);
	
	return 0;
}

File: VB/StringLength.vb
Imports System

Module StringLength
	Sub Main()
		Dim name as String
		Dim l as Integer
		Console.write("Pelase enter your name: ")
		name = Console.ReadLine()
		
		l = name.length()
		
		Console.write("Your name contains " & l & " characters.")
	End Sub
End Module